package.json

```diff
@@ -54,7 +54,7 @@
   "dependencies": {
     "@reach/accordion": "^0.17.0",
     "@reach/auto-id": "^0.17.0",
-    "@reach/dialog": "^0.18.0",
+    "@reach/dialog": "^0.17.0",
```

Comment:
Did we need to downgrade this dependency for this change?

Reply:
This recent upgrade broke Storybook, will create an issue to check it out!